docs/versioning.md

```diff
@@ -0,0 +1,97 @@
+# ONNX versioning
+
+This document describes the rules for versioning ONNX. Like the rest of the ONNX
+specification, MUST, SHOULD et al are used consistent with [RFC2119](https://tools.ietf.org/html/rfc2119).  
+
+## Versioning Principles
+
+ONNX defines versioning policy and mechanism for three classes of entities:
```

Comment:
Are versions applied across profiles (i.e. ONNX and ONNX-ML) or each profile has it's own versioning? Can users expect that ONNX-ML V1.8 is a superset of ONNX V1.8? Would we ever allow ONNX-ML V1.9 to be a superset of ONNX V1.5?